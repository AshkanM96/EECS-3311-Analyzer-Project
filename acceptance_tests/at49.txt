sum
open_set_enumeration
integer_constant (1)
integer_constant (1)
integer_constant (1)
integer_constant (1)
integer_constant (1)
integer_constant (1)
close_set_enumeration
evaluate
type_check
close_set_enumeration
open_set_enumeration
close_set_enumeration
reset
reset
sum
integer_constant (1)
evaluate
reset
sum
open_set_enumeration
integer_constant (1)
add
integer_constant (1)
integer_constant (0)
multiply
integer_constant (0)
integer_constant (1000)
integer_constant (-1)
close_set_enumeration
evaluate
