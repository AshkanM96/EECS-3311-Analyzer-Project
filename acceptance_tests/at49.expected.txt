  Expression currently specified: ?
  Report: Expression is initialized.
->sum
  Expression currently specified: (+ ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: (+ {?})
  Report: OK.
->integer_constant(1)
  Expression currently specified: (+ {1, ?})
  Report: OK.
->integer_constant(1)
  Expression currently specified: (+ {1, 1, ?})
  Report: OK.
->integer_constant(1)
  Expression currently specified: (+ {1, 1, 1, ?})
  Report: OK.
->integer_constant(1)
  Expression currently specified: (+ {1, 1, 1, 1, ?})
  Report: OK.
->integer_constant(1)
  Expression currently specified: (+ {1, 1, 1, 1, 1, ?})
  Report: OK.
->integer_constant(1)
  Expression currently specified: (+ {1, 1, 1, 1, 1, 1, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: (+ {1, 1, 1, 1, 1, 1})
  Report: OK.
->evaluate
  Expression currently specified: (+ {1, 1, 1, 1, 1, 1})
  Report: 1
->type_check
  Expression currently specified: (+ {1, 1, 1, 1, 1, 1})
  Report: (+ {1, 1, 1, 1, 1, 1}) is type-correct.
->close_set_enumeration
  Expression currently specified: (+ {1, 1, 1, 1, 1, 1})
  Report: Error (Expression is already fully specified).
->open_set_enumeration
  Expression currently specified: (+ {1, 1, 1, 1, 1, 1})
  Report: Error (Expression is already fully specified).
->close_set_enumeration
  Expression currently specified: (+ {1, 1, 1, 1, 1, 1})
  Report: Error (Expression is already fully specified).
->reset
  Expression currently specified: ?
  Report: OK.
->reset
  Expression currently specified: ?
  Report: Error (Initial expression cannot be reset).
->sum
  Expression currently specified: (+ ?)
  Report: OK.
->integer_constant(1)
  Expression currently specified: (+ 1)
  Report: OK.
->evaluate
  Expression currently specified: (+ 1)
  Report: Error (Expression is not type-correct).
->reset
  Expression currently specified: ?
  Report: OK.
->sum
  Expression currently specified: (+ ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: (+ {?})
  Report: OK.
->integer_constant(1)
  Expression currently specified: (+ {1, ?})
  Report: OK.
->add
  Expression currently specified: (+ {1, (? + nil)})
  Report: OK.
->integer_constant(1)
  Expression currently specified: (+ {1, (1 + ?)})
  Report: OK.
->integer_constant(0)
  Expression currently specified: (+ {1, (1 + 0), ?})
  Report: OK.
->multiply
  Expression currently specified: (+ {1, (1 + 0), (? * nil)})
  Report: OK.
->integer_constant(0)
  Expression currently specified: (+ {1, (1 + 0), (0 * ?)})
  Report: OK.
->integer_constant(1000)
  Expression currently specified: (+ {1, (1 + 0), (0 * 1000), ?})
  Report: OK.
->integer_constant(-1)
  Expression currently specified: (+ {1, (1 + 0), (0 * 1000), -1, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: (+ {1, (1 + 0), (0 * 1000), -1})
  Report: OK.
->evaluate
  Expression currently specified: (+ {1, (1 + 0), (0 * 1000), -1})
  Report: 0
